compiler/rustc_errors/src/annotate_snippet_emitter_writer.rs

```diff
@@ -526,12 +515,6 @@ impl AnnotateSnippetEmitter {
             }
         }
 
-        // FIXME: This hack should be removed once annotate_snippets is the
-        // default emitter.
-        if suggestions_expected > 0 && report.is_empty() {
-            group = group.element(Padding);
-        }
-
```

Comment:
I would move this change (and anything related) to its own commit after the change to make `annotate-snippets` the default. This will make it so that the switch to `annotate-snippets` shows now visible test changes (I think).